src/client/Microsoft.Identity.Client/Platforms/iOS/AuthenticationContinuationHelper.cs

```diff
@@ -57,8 +58,21 @@ public static bool IsBrokerResponse(string sourceApplication)
         /// <param name="url"></param>
         public static void SetBrokerContinuationEventArgs(NSUrl url)
         {
+            // url should always be present
```

Comment:
I still don't understand when this url can ever be null. If it is an error, shouldn't we expose it to the user?